lib/sorcery.rb

```diff
@@ -1,5 +1,13 @@
 require 'sorcery/version'
 
+if RUBY_VERSION < '2.0.0'
+  warn '[DEPRECATION] Sorcery 0.10.0 will deprecate Ruby 1.9.3 and lower, please upgrade to Ruby 2.0 or higher.'
+end
+
+if Rails::VERSION < '4.0.0'
```

Comment:
The logs say there occur`NoMethodError: undefined method `spec' for nil:NilClass` errors while installing gems.

I've experienced the same issue before because of too old versions of bundler.
https://github.com/travis-ci/travis-ci/issues/3531

However, `gem update bundler` in `.travis.yml` should solve this issue. I don't know why `before_install` is not getting executed in your build though. TravisCi config does not contain `before_install` either.